pixels-daemon/pom.xml

```diff
@@ -105,6 +117,48 @@
             <artifactId>log4j-slf4j-impl</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
We have already added junit and junit.platform to every module in the parent pom. Please remove all junit related dependency here, and use junit instead of junit-jupiter in the test code.